modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/data/engine/content/type/DLDataDefinitionContentType.java

```diff
@@ -41,6 +41,11 @@
 )
 public class DLDataDefinitionContentType implements DataDefinitionContentType {
 
+	@Override
+	public boolean allowEmptyDataDefinition() {
```

Comment:
The commit message is different from what you're trying to do here. Whats the right one?

Reply:
@jeyvison my bad. I implemented the wrong method here :sweat_smile: